src/store/modules/backend.js

```diff
@@ -64,7 +64,7 @@ export default {
       commit('tipsReloading', args);
       const value = (await Promise.all(
         times(tipsPageCount[args], (p) => Backend.getCacheTips(p + 1, ...args)),
-      )).flat();
+      )).flat().filter((p) => p);
```

Comment:
Why do we need `.filter((p) => p)` ?

Reply:
Because when you don't receive any data from backend `times` function still be called 1 time (because we request 1 page)  and will return `[null]` array, not `null` and check here https://github.com/aeternity/superhero-ui/blob/d335355d1e6ea75b4f226027187c83fca0929ce1/src/components/TipsPagination.vue#L7 will pass and you will have `Cannot get id of null` error